### ISSUE TITLE
Italy -> Audiovisual corrupted

### ISSUE THREAD

**Issue by mzeinstra** [diligentsearch/diligentsearch-website#76]
I believe I've made Italy -> Audiovisual corrupt. This occurred after build connections after a True/False node that calculates based on a numeric. I completed the branch after false and went on to working on True, I noticed that there were no names on the edges.

Please use Italy -> Audiovisual to debug this:

![screen shot 2017-06-21 at 22 43 13](https://user-images.githubusercontent.com/5627431/27405816-55ed7e62-56d3-11e7-8fb7-8d94abe6a879.png)

**Comment by mzeinstra** [comment]
I believe this to be the same issue as #74

**Comment by arnaud-brun** [comment]
I succeeded in realization of this diagram, but you will need to pull latest version.

I have updated a bit the application, a targeted node had been defined twice. This causes the targeted node to display 'click to edit', despite the data was present. Editing this node leads to a corruption (same ID used, etc...).

This is due to the target generation of a node.
This generation didn't take care if a node had already been defined, which is the case in your situation.

I have put a guard on it. Such issue could have been raised by any kind of node having targets manually set. Let me know if the problem persists.
